libs/common/src/assets/i18n/en.json

```diff
@@ -173,20 +174,21 @@
 	"billing.invalid_vat": "Please add the country code. e.g: ESB000000N",
 	"billing.month": "Month",
 	"billing.monthly_estimation.description": "The estimated cost provided is based on the defined values and is subject to change if these values are modified.",
-	"billing.monthly_estimation.title": "Overconsumption is estimated to cost",
+	"billing.monthly_estimation.title": "Monthly consumption is estimated to cost",
 	"billing.my-subscription": "My subscription",
 	"billing.next-payment": "Next payment",
 	"billing.not_company": "I'm not a company",
+	"billing.on_premise_cost": "With <b>NucliaDB On-prem</b> does not carry any extra cost.",
```

Comment:
as far as I know, `<b>` is kinda deprecated (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b). you should use `<strong>` instead

Reply:
I didn't remember, fixed